seed/views/00901_team_case_overview.js

```diff
@@ -0,0 +1,26 @@
+exports.seed = function (knex, promise) {
+  var sql = `CREATE VIEW app.team_case_overview
+  WITH SCHEMABINDING
+  AS
+  SELECT
+      MAX(CONCAT(om.forename, ' ', om.surname)) AS name
+    , MAX(om.grade_code) AS grade_code
+    , MAX(w.total_cases) AS total_cases
+    , MAX(wpc.available_points) AS available_points
+    , MAX(wpc.total_points) AS total_points
+    , MAX(wpc.contracted_hours) AS contracted_hours
+    , MAX(wpc.reduction_hours) AS reduction_hours
+    , MAX(t.id) AS team_id
```

Comment:
The case-progress views use `id` rather that `team_id`, `region_id` etc. This allows the get-case-progress to become more generic.  Can you change this to use `id` as well.